v3/src/components/graph/adornments/adornments-base-store.ts

```diff
@@ -17,6 +19,18 @@ export const AdornmentsBaseStore = types.model("AdornmentsBaseStore", {
   showMeasureLabels: false,
   showSquaresOfResiduals: false
 })
+.views(self => ({
+  isShowingAdornment(type: string) {
+    return !!self.adornments.find(a => a.type === type)?.isVisible
+  },
+  get activeUnivariateMeasures() {
+    return self.adornments.filter(adornment => adornment.isUnivariateMeasure && adornment.isVisible) as
+            IUnivariateMeasureAdornmentModel[]
+  },
+  findAdornmentOfType<T extends IAdornmentModel = IAdornmentModel>(type: string): T | undefined {
+    return self.adornments.find(adornment => adornment.type === type) as T
+  }
+}))
```

Comment:
Note that these are not new methods. They've just moved so that they can be more easily called internally.